packages/geolocator/example/tizen/Runner.csproj.user

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup/>
+  <PropertyGroup>
+    <FlutterEmbeddingPath>/home/boram/Work/f-project/flutter-tizen/embedding/csharp/Tizen.Flutter.Embedding.csproj</FlutterEmbeddingPath>
+  </PropertyGroup>
+</Project>
```

Comment:
newline also

Reply:
done

Reply:
flutter-tizen removes newlines in this file :cry:

Reply:
Yes, It seems that the `XMLDocument` of `package:xml` writes the file without EOF newline. :(

Reply:
I made a patch https://github.com/flutter-tizen/flutter-tizen/pull/185

Reply:
Hey guys. Just don't check out the .user file.

@bbrto21 Maybe your project had been created before https://github.com/flutter-tizen/flutter-tizen/pull/178 was merged. Please run `flutter-tizen create` again and make sure the `.user` file is ignored by git.